File: beginner/1072.py
n = int(input())
i = 0
numbers = []
iN = 0
out = 0

while i < n:
	numbers.append(int(input()))
	i += 1
	
for x in numbers:
	if x >= 10 and x <= 20:
		iN += 1
	else:
		out += 1
		
print(f"{iN} in")
print(f"{out} out")

File: beginner/1072.c
# include <stdio.h>

int main (){
	int i, n, numbers[10000] = {}, in = 0, out = 0;
	
	scanf("%i", &n);
	
	for( i = 0 ; i < n ; ++i)
		scanf("%i", &numbers[i]);
	
	for( i = 0 ; i < n ; ++i ){
		if(numbers[i] >= 10 && numbers[i] <= 20)
			++in;
		else
			++out;
	}
	
	printf("%i in\n", in);
	printf("%i out\n", out);
	
	return 0;
}
